pkg/datapath/linux/probes/probes.go

```diff
@@ -133,8 +133,10 @@ func NewProbeManager() *ProbeManager {
 	newProbeManager := func() {
 		var features Features
 		out, err := exec.WithTimeout(
-			defaults.ExecTimeout, "bpftool", "-j", "feature").CombinedOutput(
-			log, true)
+			defaults.ExecTimeout, "bpftool", "-j", "feature",
+			"probe", "filter_out",
+			"'\\(trace\\|perf\\|write_user\\)'",
```

Comment:
Are perf-related helpers printing kernel logs as well?
Filtering out on `perf` will also remove probe results for `BPF_MAP_TYPE_PERF_EVENT_ARRAY` and related helpers, is that desirable?

Reply:
Hmm... I checked and filtering out `perf` doesn't seem to be necessary.

So `bpftool -j feature probe filter_out '\(trace\|write_user\)'` works. I will change that. Thanks!

Reply:
done